src/Bridge/Doctrine/Orm/Extension/PaginationExtension.php

```diff
@@ -143,7 +144,16 @@ public function supportsResult(string $resourceClass, string $operationName = nu
      */
     public function getResult(QueryBuilder $queryBuilder, string $resourceClass = null, string $operationName = null, array $context = [])
     {
-        $doctrineOrmPaginator = new DoctrineOrmPaginator($queryBuilder, $this->useFetchJoinCollection($queryBuilder, $resourceClass, $operationName));
+        $parts = $queryBuilder->getDQLPart('join');
+        $rootAlias = $queryBuilder->getRootAliases();
+        $query = $queryBuilder->getQuery();
+
+        // Only one alias, without joins, disable the DISTINCT on the COUNT
```

Comment:
```suggestion
        // Only one alias, without join, disable the DISTINCT on the COUNT
```

Reply:
don't agree

Reply:
Fun English fact: Both "join" and "joins" are correct here! :see_no_evil: